### PR TITLE
Update Notifications Page

### PR DESCRIPTION
**Pull request by lbk-fishtown**
## Description & motivation
There is an existing notifications page for Slack, but not one for email which is a more common question.

I have updated and renamed the existing Slack page to be a generic title, and added a section for Email.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [x] The page has been added to `website/sidebars.js`
- [x] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [x] The page has been removed from `website/sidebars.js`
- [x] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+title: "Configuring Notifications"
+---
+
+### Overview
+
+Setting up notifications in dbt Cloud will allow you to receive alerts via Email or a chosen Slack channel when a job run succeeds, fails, or is cancelled.
+
+### Email
+
+To set up email noticiations first navigate to your **Profile** (found in the top right hand side of dbt Cloud) and the select **Notifications**.
```
Comment (amychen1776) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md]: You can also go to Account Settings to set up notifications for your entire team. Did you want to also provide this path?

Comment (lbk-fishtown) [reply website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md]: Thanks Amy! Just made a commit to include that aspect.

Comment (lbk-fishtown) [reply website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md]: @amychen1776 if you have a moment to review this change would be awesome :)

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+title: "Configuring Notifications"
+---
+
+### Overview
+
+Setting up notifications in dbt Cloud will allow you to receive alerts via Email or a chosen Slack channel when a job run succeeds, fails, or is cancelled.
+
+### Email
+
+There are two options for setting up email notifications. As a **user** you can set up email notifications for yourself under your Profile. As an **admin** you can setup notifications on your team members behalf.
```
Comment (amychen1776) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md]:
```suggestion
There are two options for setting up email notifications. As a **user**, you can set up email notifications for yourself under your Profile. As an **admin**, you can set up notifications on your team members' behalf.
```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+* **As a user:** To set up email notifications first navigate to your **Profile** (found in the top right hand side of dbt Cloud) and the select **Notifications**.\
```
Comment (amychen1776) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md]:
```suggestion
* **As a user:** Navigate to your **Profile** (found in the top right-hand side of dbt Cloud) and select **Notifications**.
```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+\
+Next select **Edit** and select the type of Notification (Succeeds, Fails or Is Cancelled) for each Job for which you would like to be notified.
```
Comment (amychen1776) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md]:
```suggestion
Next select **Edit** and select the type of Notification (Succeeds, Fails, or Is Cancelled) for each Job for which you would like to be notified.
```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+* **As an admin:** To set up email notifications navigate to Account Settings > Notifications.\
```
Comment (amychen1776) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md]:
```suggestion
* **As an admin:**  Navigate to Account Settings (in the hamburger menu) > Notifications.
```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+\
```
Comment (amychen1776) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md]:
```suggestion

```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+Select the User you'd like to set notifications for. Select **Edit** and select the type of Notification (Succeeds, Fails or Is Cancelled) for each Job for which they will be notified.
```
Comment (amychen1776) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-notifications.md]:
```suggestion
Select the User you'd like to set notifications for. Select **Edit** and select the type of Notification (Succeeds, Fails, or Is Cancelled) for each Job for which they will be notified.
```

**Review by amychen1776 — CHANGES_REQUESTED** [review]
Thank you so much for working on this! I added some suggestions after taking a look at the deploy preview.

**Comment by lbk-fishtown** [comment]
Hi @amychen1776 I have incorporated the suggestions you made, let me know what you think. 

Thank you so much for looking at this! I don't know how I missed your suggestions back in July - I need to check my email more closely for the notifications next time - sorry!

**Comment by amychen1776** [comment]
@lbk-fishtown Curious if you still wanted to merge this or?

**Pull request merged**